src/components/ThemeProvider/tests/ThemeProvider.test.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 // eslint-disable-next-line no-restricted-imports
 import {mountWithAppProvider} from 'test-utilities/legacy';
+import {mount} from 'enzyme';
```

Comment:
Having to import mount and mountWithAppProvider from two separate places feels odd considering how similar they are. Can you update `test-utilities/legacy` so it reexports `mount` from enzyme and then you can do

```js
import {mount, mountWithAppProvider} from 'test-utilities/legacy';
```


We follow a similar pattern in `test-utilities` already

Reply:
Following up from this, It'd be just as easy to use the new react-testing library by importing  `import {mount} from 'test-utilities';` and it'd mean you're not using a library we want to move away from. Could you do that instead?